src/packages/end-to-end/src/__tests__/basic-query.test.ts

```diff
@@ -0,0 +1,26 @@
+import request from 'supertest-graphql';
+import gql from 'graphql-tag';
+
+import { Album } from '../types';
+import { config } from '../config';
+import { resetDatabase } from '../utils';
+
+describe('basic query', () => {
+	beforeEach(resetDatabase);
+
+	test('should get albums', async () => {
+		const { data } = await request<{ albums: Album[] }>(config.baseUrl)
+			.query(
+				gql`
+					query {
+						albums {
+							id
+						}
+					}
+				`
+			)
+			.expectNoErrors();
+
+		expect(data?.albums).toHaveLength(347);
```

Comment:
Similar to the above, looks like we're expecting this to just build and run once. If so, all good.

Reply:
yeah same as above 👍